docs/samples/v1beta1/transformer/torchserve_image_transformer/README.md

```diff
@@ -16,12 +16,20 @@ kubectl port-forward pods/{name-of-predictor-pod} 8081:8080
 ```
 Since the predictor pod will expose 8080, pick another port to use with localhost. Here, 8081 is used.
 
+*Note that if you use KServe 0.11 or later, you can skip this step and add `--use_ssl` flag to the command below.*
```

Comment:
Do we need to add the note for the environment variable you need to set to make the gRPC https call ?